test/Benchmarks/Instrumentation/InstrumentedAspNetCoreBenchmark.cs

```diff
@@ -27,22 +31,32 @@ namespace Benchmarks.Instrumentation
     public class InstrumentedAspNetCoreBenchmark
     {
         private const string LocalhostUrl = "http://localhost:5050";
+        private const string ResourceName = "aspnetcore-service-example";
+        private const string SourceName = "aspnetcore-test";
 
         private HttpClient client;
         private LocalServer localServer;
+        private TracerProvider tracerProvider;
 
         [GlobalSetup]
         public void GlobalSetup()
         {
             this.localServer = new LocalServer(LocalhostUrl, true);
             this.client = new HttpClient();
+
+            this.tracerProvider = Sdk.CreateTracerProviderBuilder()
```

Comment:
LocalServer(..true) will do this automatically. We dont need to do this. But I think the right way is to do it here, and remove the logic from LocalServer.

Reply:
wait...we did have the correct benchmark...

I forgot about that! I will close the pr, since this will duplicate code and the other way it is already reusing.